src/style.css

```
@@ -1,3 +1,71 @@
+html {
+  --blue-color: #00acdc;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.container {
+  padding: 0 50px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo img {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+ul {
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+li {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.menu__link {
+  display: inline-block;
+  height: var(--header-height);
+  line-height: var(--header-height);
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #000;
+  font-size: 12px;
+}
+
+.menu__link:hover {
+  color: var(--blue-color);
+}
+
+.menu__list :nth-child(n + 2) {
+  margin-left: 20px;
+}
```

```suggestion
.nav__item:not(:first-child) {
  margin-left: 20px;
}
```

we can simplify it a bit